queries/generators/belgium.rq

```diff
@@ -56,4 +50,4 @@ UNION
 
   SERVICE wikibase:label { bd:serviceParam wikibase:language "en,nl,fr,de" }
 }
-ORDER BY ?type ?orgLabel
+ORDER BY ?typeLabel ?type
```

Comment:
```suggestion
ORDER BY ?typeLabel ?orgLabel
```
I buy that sorting on the label for type may make more sense, but then the type adds no value at all (as we will never have several types for each type label). Let's keep the alphabetical sorting of the orgs within each type.